geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloPartitioningTest.scala

```diff
@@ -33,6 +32,8 @@ import scala.collection.JavaConversions._
 @RunWith(classOf[JUnitRunner])
 class AccumuloPartitioningTest extends Specification with TestWithDataStore {
 
+  // note: using `Seq.foreach; ok` instead of `foreach(Seq)` shaves several seconds off the time to run this test
```

Comment:
Interesting... any idea why?!

Reply:
maybe because it builds up complex matchers and tracks the results?